### PR TITLE
clarifications and samples

### PR DESCRIPTION
**Pull request by bkimminich**
- clarified who can wear plate armor
- introduced XP abbreviation
- added some alignment limitations (only good paladins, no good rogues, no evil halflings)
- added shield as another armor sample

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -1,14 +1,16 @@
-# The EverCraft Kata
+# The EverCraft Kata [![Dependency Status](https://www.versioneye.com/user/projects/5630a4a536d0ab0016001a17/badge.svg?style=flat)](https://www.versioneye.com/user/projects/5630a4a536d0ab0016001a17)
```
Comment (walterg2) [inline README.markdown]: Let's remove the dependency status from here. I like the idea and will implement it via the D&D account.

File: README.markdown [hunk 2]
```diff
@@ -273,15 +286,57 @@ As a character I want to be able to have items that enhance my capabilities so t
...
+|   __3__   |   3,000    |   __8__   |   28,000   |  __13__   |  78,000    |  __18__   |  153,000   |
+|   __4__   |   6,000    |   __9__   |   36,000   |  __14__   |  91,000    |  __19__   |  171,000   |
+|   __5__   |  10,000    |  __10__   |   45,000   |  __15__   | 105,000    |  __20__   |  190,000   |
 
 ## Bonus Iteration - Battle Grid
 
 Build your own features here.  Multiple characters can be on a grid-based map.  Each square on the map had terrain
 that impacts the occupant or opponents attacking into it.  Characters can move and weapons have ranges.
+
+# Sample Implementations
```
Comment (walterg2) [inline README.markdown]: This is something we do not want to have in the readme for a few reasons:

1- We don't want to have users "sampling" existing solutions to figure out the various "gotchas" that are in the kata on purpose.
2- We also take this README and convert it to PDF so people in our workshop can have a paper copy to work from when an internet connection is not very good.

All in all, could you remove this section from the document?

File: README.markdown [hunk 3]
```diff
@@ -273,15 +286,57 @@ As a character I want to be able to have items that enhance my capabilities so t
...
+> As a character I want to be able to wear a ring of protection so that I can be protected from attack
 
   - adds +2 to armor class
 
-As a character I want to be able to war a belt of giant strength so that I can be even stronger in combat
+> As a character I want to be able to war a belt of giant strength so that I can be even stronger in combat
 
   - add +4 to Strength Score
+  
+## Iteration 5 - Improvements to existing features
```
Comment (walterg2) [inline README.markdown]: I like these additions, but just a fair warning... Most pairs do not complete Iteration 4 (most only making it to the end of Iteration 2 or the beginning of Iteration 3) in an 8 hour period, so don't expect many people to make it to this part before the end of Clean Code Days. :-)

**Comment by bkimminich** [comment]
- figured the badge issue just before your comment... :smile:
- sample implementation section is gone

**Comment by bkimminich** [comment]
I removed the "critter" term as the idea of non-character creatures is not in the kata anyway. Might confuse people who are not into RPGs so much...

**Comment by bkimminich** [comment]
Hi @walterg2, I just checked your slide deck associated with this kata, and it seems my 5th iteration would get those slides out of sync, which I never intended. As you mentioned that most pairs don't make it to iteration 4 in the first place, I could just move my proposed changes down to the "Bonus Iteration" as well. Just let me know how you prefer to have it.

**Comment by walterg2** [comment]
Hey @bkimminich, since there's a bit of uncertainty in where the clarifications fit in, would you be willing to pull them from this PR and add a second one just for that? It will allow us to get the changes that everyone's good with in right away without keeping this PR dragging along while we discuss the new features.

What do you think?

**Comment by bkimminich** [comment]
Good idea. I moved the Iteration 5 content to a separate branch https://github.com/bkimminich/EverCraft-Kata/tree/additional-features. PR should be cleared of changes under discussion now, but if you find something else, just let me know.

**Comment by bkimminich** [comment]
Hi @walterg2, anything left for me to change before you can merge this PR? I'm just asking if I should point my Clean Code Days-Dojo participants to my fork or the original repo. Would prefer the latter of course.

**Pull request merged**